BanyanArrays/test/Project.toml

```diff
@@ -4,7 +4,6 @@ Banyan = "706d138b-e922-45b9-a636-baf8ae0d5317"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FilePathsBase = "48062228-2e41-5def-b9a4-89aafe57970f"
-HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
```

Comment:
this is removing HDF5 from the test directory...
Is HDF5 still required in `BanyanArrays/Project.toml`?